challenge.md

```diff
@@ -31,20 +31,20 @@ Open up this file in your text editor, and write your answers below (where indic
 
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
- <!-- Answer Starts Here -->
- 
+ <!-- absolute. it begins with a leading slash and is relative to the rood directory-->
+
  <!-- Answer Ends Here -->
 
 2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
 
- <!-- Answer Starts Here -->
- 
+ <!-- Absolute. Because it's not on our computer, we need the whole path to get to it. -->
```

Comment:
Basically. All URLs are absolute paths by definition. And that makes sense, because we want different people (at different locations) to all be able to find the exact same file or resource.